istioctl/pkg/auth/upgrader.go

```diff
@@ -162,6 +207,36 @@ func (ug *Upgrader) convert(authzPolicies *model.AuthorizationPolicies) error {
 	return nil
 }
 
+// convertClusterRbacConfig converts ClusterRbacConfig to AuthorizationPolicy.
+func (ug *Upgrader) convertClusterRbacConfig(authzPolicies *model.AuthorizationPolicies) error {
+	clusterRbacConfig := authzPolicies.GetClusterRbacConfig()
+	if clusterRbacConfig == nil {
+		return fmt.Errorf("no ClusterRbacConfig found")
+	}
+	rootNamespace, err := ug.getIstioRootNamespace()
+	if err != nil {
+		return fmt.Errorf("failed to get Istio root namespace: %s", err)
+	}
+	type RbacTmplData struct {
```

Comment:
You could also do:

```golang
templateData := struct{
    RootNamespace:: rootNamespace,
}
```

Reply:
Is it some Go idiom? Doesn't seem to work for me. I think we can do if you think it's cleaner.

```
templateData := struct {
	RootNamespace string
}{
	RootNamespace: rootNamespace,
}
```
Wdyt?

Reply:
lgtm

Reply:
Or you could just use a map e.g.

```golang
data := map[string]string{
    "RootNamespace": rootNamespace
}
```

Reply:
The map looks cleaner, thanks